### DISCUSSION TITLE
usability, code correction, code generation

### DISCUSSION THREAD

**Discussion by DarklinuxFr** [community/community#69597] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Hello, I am quite satisfied with Copilot, but this one, although interesting, is difficult to invoke. Why not integrate the chat window in a corner during a future relesase?
The code correction is interesting, but it is not carried out in the active window. Why wouldn't the chatbot, after an accepted suggestion, do it again in the workspace?
To conclude: Why, with this chat window, not automatically generate this or that skeleton file, which Copilot then suggested we fill in according to our context?

**Comment by moayad-star** [comment]
They will certainly add this feature if it is of benefit to the developers, and you can use it in the usual way until any update is published or a trial is released in the future. Just stay tuned for updates to receive notification.If the developers publish any update
